service/src/main/resources/quips/complibot/compliments.json

```diff
@@ -229,5 +240,126 @@
     "tags":[],
     "targetableText":"If there was a zombie apocalypse, I'd want resourceful people like [TARGET] in my camp.",
     "targetableSsml":"If there was a zombie apocalypse, I'd want resourceful people like [TARGET] in my camp."
+  },
+  {
+    "quipGroup":"INFECTIOUS_SMILE",
+    "text":"We better call the doctor, because your smile is infectious.",
```

Comment:
I thought about that originally, but ended up going with compliment. It's probably more important right now to have more content for the primary type of use. Since asking CompliBot for an insult is currently completely undocumented haha